vpr/src/analytical_place/partial_placement.cpp

```diff
@@ -8,8 +8,29 @@
 #include "partial_placement.h"
 #include <cmath>
 #include <cstddef>
+#include <limits>
 #include "ap_netlist.h"
 
+double PartialPlacement::get_hpwl(const APNetlist& netlist) const {
+    double hpwl = 0.0;
+    for (APNetId net_id : netlist.nets()) {
```

Comment:
Are the ignored nets removed from the ap netlist so there's no need to check for them here?

Reply:
That is correct. This HPWL is not meant to be accurate, its meant to track how the Global Placer is converging. If we used a more accurate HPWL which knew more information about the problem than the solver, we may never converge I think.